pkg/kn/commands/service/configuration_edit_flags.go

```
@@ -333,13 +334,45 @@ func (p *ConfigurationEditFlags) Apply(
 		} else if cmd.Flags().Changed("scale-min") {
 			return fmt.Errorf("only --scale or --scale-min can be specified")
 		} else {
-			err = servinglib.UpdateMaxScale(template, p.Scale)
-			if err != nil {
-				return err
-			}
-			err = servinglib.UpdateMinScale(template, p.Scale)
-			if err != nil {
-				return err
+			if !strings.Contains(p.Scale, "..") {
+				scaleInt, _ := strconv.Atoi(p.Scale)
+				err = servinglib.UpdateMaxScale(template, scaleInt)
+				if err != nil {
+					return err
+				}
+				err = servinglib.UpdateMinScale(template, scaleInt)
+				if err != nil {
+					return err
+				}
+			} else if len(p.Scale) == 4 {
+				scaleParts := strings.Split(p.Scale, "..")
+				scaleMin, _ := strconv.Atoi(scaleParts[0])
```

Same here... don’t ignore errors